dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileVisitor.java

```diff
@@ -89,9 +87,16 @@ public void visitMethodCallExpression(final MethodCallExpression call) {
             testLine = call.getLastLineNumber() + offset;
             if (call.getArguments() instanceof ArgumentListExpression) {
                ArgumentListExpression arguments = (ArgumentListExpression) call.getArguments();
-               int propertiesLine = parseTaskWithPotentialSystemProperties(arguments);
+               int propertiesLine = parseTaskWithPotentialSystemProperties(arguments, testExecutionProperties);
                testSystemPropertiesLine = propertiesLine;
             }
+         } else if (call.getMethodAsString().equals("integrationTest")) {
+            testLine = call.getLastLineNumber() + offset;
+            if (call.getArguments() instanceof ArgumentListExpression) {
+               ArgumentListExpression arguments = (ArgumentListExpression) call.getArguments();
+               int propertiesLine = parseTaskWithPotentialSystemProperties(arguments, integrationtestExecutionProperties);
```

Comment:
Why don't you assign ` integrationTestSystemPropertiesLine`  directly?

Reply:
This is refactored